build.gradle

```diff
@@ -37,11 +38,18 @@ dependencies {
     implementation(group: 'org.springframework.boot', name: 'spring-boot-starter-actuator', version: springBootVersion)
     implementation(group: 'org.springframework.boot', name: 'spring-boot-devtools', version: springBootVersion)
     implementation(group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: springBootVersion)
+    implementation(group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.15.2')
```

Comment:
we are adding primary dependencies here that we do not actually depend on - if we need to bump the version of transitive libraries, we should do that as a constraint on the primary dependency that pulls them in - this is not the right way to bump these versions.  See https://docs.gradle.org/7.3/userguide/dependency_constraints.html

Reply:
Updated version by leveraging spring-dependency-management plugin variables and via gradle constraints.